packages/x-data-grid-premium/src/components/GridFooterCell.tsx

```diff
@@ -10,10 +11,10 @@ const GridFooterCellRoot = styled('div', {
   name: 'MuiDataGrid',
   slot: 'FooterCell',
   overridesResolver: (_, styles) => styles.footerCell,
-})<{ ownerState: OwnerState }>(({ theme }) => ({
-  fontWeight: theme.typography.fontWeightMedium,
-  color: (theme.vars || theme).palette.primary.dark,
-}));
+})<{ ownerState: OwnerState }>({
+  fontWeight: vars.typography.fontWeight.medium,
+  color: 'red', // FIXME: (theme.vars || theme).palette.primary.dark,
```

Comment:
@KenanYusuf Which color did you have in mind for this case?

Reply:
```diff
diff --git a/packages/x-data-grid/src/components/containers/GridRootStyles.ts b/packages/x-data-grid/src/components/containers/GridRootStyles.ts
index ed1f2ae28..691332934 100644
--- a/packages/x-data-grid/src/components/containers/GridRootStyles.ts
+++ b/packages/x-data-grid/src/components/containers/GridRootStyles.ts
@@ -783,6 +783,7 @@ function transformMaterialUITheme(t: Theme) {
     [vars.colors.background.overlay]: t.palette.background.paper,
     [vars.colors.foreground.base]: t.palette.text.primary,
     [vars.colors.foreground.muted]: t.palette.text.secondary,
+    [vars.colors.foreground.accent]: t.palette.primary.dark,
     [vars.colors.foreground.disabled]: t.palette.text.disabled,
     [vars.colors.interactive.hover]: t.palette.action.hover,
     [vars.colors.interactive.hoverOpacity]: t.palette.action.hoverOpacity,
diff --git a/packages/x-data-grid/src/constants/cssVariables.ts b/packages/x-data-grid/src/constants/cssVariables.ts
index c0ea70233..f0f05a19a 100644
--- a/packages/x-data-grid/src/constants/cssVariables.ts
+++ b/packages/x-data-grid/src/constants/cssVariables.ts
@@ -32,6 +32,7 @@ export const vars = {
     foreground: {
       base: '--DataGrid-t-colors-foreground-base',
       muted: '--DataGrid-t-colors-foreground-muted',
+      accent: '--DataGrid-t-colors-foreground-accent',
       disabled: '--DataGrid-t-colors-foreground-disabled',
     },
```